core/server/data/migrations/versions/2.2/1-add-sessions-table.js

```diff
@@ -0,0 +1,35 @@
+const common = require('../../../../lib/common');
```

Comment:
General: I guess the plan is to merge all pull requests with migrations till Tuesday and release Ghost 2.2 as a database migration release? cc @kevinansfield cc @allouis

Reply:
I was under that impression yeah :+1:

Reply:
Ok :) Then we just need to ensure that all the PR's with migrations are mergeable and get reviewed :)